modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -1648,27 +1649,15 @@ private void _validateListTypeDefinitionId(
 
 	private void _validateLocalized(
 			boolean localized, ObjectDefinition objectDefinition,
+			ObjectField objectField,
 			ObjectFieldBusinessType objectFieldBusinessType, boolean required)
 		throws PortalException {
 
 		if (!localized) {
 			return;
 		}
 
-		String objectFieldBusinessTypeName = objectFieldBusinessType.getName();
-
-		if ((!FeatureFlagManagerUtil.isEnabled(
-				objectDefinition.getCompanyId(), "LPD-32050") &&
-			 !objectFieldBusinessTypeName.equals(
-				 ObjectFieldConstants.BUSINESS_TYPE_LONG_TEXT) &&
-			 !objectFieldBusinessTypeName.equals(
-				 ObjectFieldConstants.BUSINESS_TYPE_RICH_TEXT) &&
-			 !objectFieldBusinessTypeName.equals(
-				 ObjectFieldConstants.BUSINESS_TYPE_TEXT)) ||
-			(FeatureFlagManagerUtil.isEnabled(
-				objectDefinition.getCompanyId(), "LPD-32050") &&
-			 !objectFieldBusinessType.isLocalizable())) {
-
```

Comment:
I have concerns about removing this condition. This method validates the localized property via both API and UI. 

With the changes in `objectFieldBusinessType.isLocalizable(objectField)`/ `supportLocalization(objectField)`, it appears that all field types will return `false` when the feature flag is disabled. 

This could lead to unintended errors, even for text-based fields that were previously allowed without the feature flag. Could you please double-check this logic and ensure it doesn't introduce a regression?

Reply:
Because of this, I kept the override returning true for fields that do not depend on the feature flag. For me, this logic is compatible with the `supportLocalization`. I will recreate all the fields of the object to ensure it keeps working.

Reply:
We discussed keeping this IF and removing the code from Text/LongText/RichText.
```
@Override
public boolean supportLocalization(ObjectField objectField) {
    return true;
}
```